README.md

```diff
@@ -87,3 +87,22 @@ When providing either of these functions, React Native Walkthrough Tooltip will
 **NOTE: This will disable and override any touch events on your child element**
 
 One possible use case for these functions would be a scenerio where you are highlighting new functionality and want to restrict a user to ONLY do a certain action when they press on an element. While perhaps uncommon, this use case was relevant for another library I am working on, so it may be useful for you. When these props are NOT provided, all touch events on children occur as expected.
+
+### TooltipChildrenConsumer
+
+[React Context](https://reactjs.org/docs/context.html) consumer that can be used to distinguish "real" children rendered inside parent's layout from their copies rendered inside tooltip's modal.
```

Comment:
I'd add a sentence at the end that just further explains the way it works, something similar to

"The duplicate child rendered in the tooltip modal is wrapped in a context Provider which has the value `tooltipDuplicate: true`, so informed decisions may be made, if necessary, based on where the child getting rendered"

Maybe a bit verbose, but I prefer to be overly explicit. Then backing it up with the code example is great!